data_structures/heap/skew_heap.py

```diff
@@ -2,9 +2,17 @@
 
 from __future__ import annotations
 
-from typing import Generic, Iterable, Iterator, TypeVar
+from abc import ABCMeta, abstractmethod
+from typing import Any, Generic, Iterable, Iterator, TypeVar
 
-T = TypeVar("T")
+
+class Comparable(metaclass=ABCMeta):
```

Comment:
Do we need this class?

Reply:
Yes, that class is used to make the generic type ` T ` comparable.
Without it, we get this `mypy` error:  `
Unsupported left operand type for > ("T")`

Reply:
I just found another solution that doesn't need the class, so I will make an update. @L3str4nge